src/lib/metricsMgr/MetricsManager.h

```diff
@@ -34,12 +34,20 @@
 /* ****************************************************************************
 *
 * Metrics - 
+*
+* NOTE
+*   METRIC_TOTAL_SERVICE_TIME is in the metrics map but excluded from
```

Comment:
Maybe we could use some syntax for those that are not indended to be printed. This would make easier debugging work, if needed. E.g. `__totalServiceTime`